guides/common/modules/con_upgrading-prerequisites.adoc

```diff
@@ -16,6 +16,9 @@ Upgrading {SmartProxy} takes approximately 10{range}30 minutes.
 
 * Ensure that you have sufficient storage space on your server.
 For more information, see {InstallingServerDocURL}Preparing_your_Environment_for_Installation_{project-context}[Preparing your Environment for Installation] in _{InstallingServerDocTitle}_ and {InstallingSmartProxyDocURL}preparing-environment-for-capsule-installation[Preparing your Environment for Installation] in _Installing {SmartProxyServer}_.
+* Ensure that you have at least the same amount of free space as that consumed by `/var/lib/pgsql/data` for PostgreSQL upgrade.
+The upgrade involves PostgreSQL 12 to PostgreSQL 13 upgrade.
```

Comment:
On this line, you are talking about two different upgrades: Project and PostgreSQL. I recommend to specify each of them to prevent confusion. This would be one way to do it:
```suggestion
Upgrading to {Project} {ProjectVersion} involves a PostgreSQL 12 to PostgreSQL 13 upgrade.
```